src/main/scala/io/iohk/ethereum/network/p2p/validators/SignedTransactionValidator.scala

```diff
@@ -0,0 +1,80 @@
+package io.iohk.ethereum.network.p2p.validators
```

Comment:
You are right, I'll create a separate package for the `validators`, I don't know why they were all there :stuck_out_tongue_winking_eye: